File: ppt.java
import java.util.Scanner;
public class ppt{
  public static boolean Checkers2(int x,int y){
    boolean who=false;
    if((x == 1) && (y==2))
      who=false;
    else if((x == 2) && (y==3))
      who=false;
    else if((x == 3) && (y==1))
      who=false;
    else
      who=true;


    return who;
  }

  public static void ppt1() {
    Scanner sc=new Scanner(System.in);
    int scoreh=0;
    int scorem=0;



    while(scoreh!=3 && scorem!=3){
      int y=(int)(Math.random() * ((3 - 1) + 1)) + 1;

      System.out.println("Insert Choice(Number)\n=================\n1.-Papel\n2.-Tijeras\n3.-Piedra");
      System.out.println(y);
      int x=sc.nextInt();

      if(x!=y){
        boolean who=Checkers2(x,y);
        if(who){
          scoreh ++;
          System.out.println("================================\nHuman WON!");
          System.out.println("Scores: \nMachine:"+scorem+"\nHuman"+scoreh+"\n================================");
        }
        else{
          scorem++;
          System.out.println("================================\nMachine WON!");
          System.out.println("Scores: \nMachine:"+scorem+"\nHuman"+scoreh+"\n================================");
        }
      }
      else{
        System.out.println("================================\nAgain!\n================================");
      }
    }
  }




  public static void main(String[] args) {
    ppt1();
  }
}

File: ppt.py
from random import randint


scoreh=0
scorem=0
win=0
machine=0
z=0
while z!=3:


    y = float(randint(1,3))
    x = float(input("1- Papel ; 2-Tijeras ; 3-Piedra"))

    if x != y:
        if (x == 1) and (y==2):
            scorem += float(1)
            print("La máquina ha ganado")
            machine+=1
        elif (x==2) and (y==3):
            scorem += float(1)
            print("La máquina ha ganado")
            machine+=1
        elif (x==3) and (y==1):
            scorem += float(1)
            print("La máquina ha ganado")
            machine+=1
        else:
            scoreh += float(1)
            win += 1
            z +=1
            print("Has ganado")
    else:
        print("Repetir")
    print(y)
input()
